StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
```

Comment:
>netcoreapp3 [](http://example.com/codeflow?start=42&length=11)

is this script cross-platform friendly or not? you have hard-coded directory separators here. 

do we care if this script if cross-platform friendly?